### ISSUE TITLE
DummyClientReplicatedMapTest.testClear

### ISSUE THREAD

**Issue by Donnerbart** [hazelcast/hazelcast#6871]
```
java.lang.AssertionError: expected:<0> but was:<1>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.client.replicatedmap.DummyClientReplicatedMapTest$7.run(DummyClientReplicatedMapTest.java:222)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:835)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:849)
	at com.hazelcast.client.replicatedmap.DummyClientReplicatedMapTest.testClear(DummyClientReplicatedMapTest.java:219)
```
https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-pr-builder/com.hazelcast$hazelcast-client/17103/testReport/junit/com.hazelcast.client.replicatedmap/DummyClientReplicatedMapTest/testClear/

**Comment by eminn** [comment]
fixed via c333fd30d020324db1aabe79544c61baa1916fbd

**Comment by Donnerbart** [comment]
I think this run is newer than the fix, isn't it? If not, just close the ticket again:
https://hazelcast-l337.ci.cloudbees.com/job/new-lab-fast-pr/132/testReport/junit/com.hazelcast.client.replicatedmap/DummyClientReplicatedMapTest/testClear/
